.betterci/ci-k8s-manifest.yml

```diff
@@ -56,33 +66,63 @@ spec:
           image: #{portal-server}
           env:
             - name: DB_SERVER
-              value: "mongo-service:27017"
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: DataBaseServer
             - name: JWT_SECRET
-              value: "litmus-portal@123"
-            - name: EXTERNAL_ADDRESS
-              value: ""
-            - name: SERVICE_ADDRESS
-              value: "http://litmusportal-server-service:9002"
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: JWTSecret
             - name: SELF_CLUSTER
               value: "false"
+            - name: AGENT_SCOPE
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: AgentScope
+            - name: AGENT_NAMESPACE
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: AgentNamespace
+            - name: LITMUS_PORTAL_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: PORTAL_SCOPE
+              value: "cluster"
             - name: SUBSCRIBER_IMAGE
-              value: #{subscriber}
-            - name: SUBSCRIBER_NAMESPACE
-              value: "litmus"
+              value: "litmuschaos/litmusportal-subscriber:ci"
```

Comment:
SUBSCRIBER_IMAGE and DEPLOYER_IMAGE should also be a template env. Right?

Reply:
Updated subscriber image, self_deployer image is not required as self_cluster isn't isn't set to true here.